### PR TITLE
wireless/bcm43xxx: add customized MAC address support

### PR DESCRIPTION
**Pull request by anchao**
## Summary

wireless/bcm43xxx: add customized MAC address support

## Impact

N/A

## Testing

bcm43013 customized MAC address set

### CODE REVIEW

File: boards/arm/stm32/emw3162/src/stm32_wlan.c [hunk 1]
```diff
@@ -126,6 +126,15 @@ void bcmf_board_setup_oob_irq(int minor, int (*func)(void *), void *arg)
     }
 }
 
+/****************************************************************************
+ * Name: bcmf_board_etheraddr
+ ****************************************************************************/
+
+bool bcmf_board_etheraddr(FAR struct ether_addr *ethaddr)
```
Comment (xiaoxiang781216) [inline boards/arm/stm32/emw3162/src/stm32_wlan.c:133]: remove FAR

File: boards/arm/stm32/photon/src/stm32_wlan.c [hunk 2]
```diff
@@ -108,6 +108,15 @@ void bcmf_board_setup_oob_irq(int minor, int (*func)(void *), void *arg)
     }
 }
 
+/****************************************************************************
+ * Name: bcmf_board_etheraddr
+ ****************************************************************************/
+
+bool bcmf_board_etheraddr(FAR struct ether_addr *ethaddr)
```
Comment (xiaoxiang781216) [inline boards/arm/stm32/photon/src/stm32_wlan.c:115]: ditto

File: boards/arm/stm32/emw3162/src/stm32_wlan.c (hunk 1 above)
Comment (anchao) [reply boards/arm/stm32/emw3162/src/stm32_wlan.c:133]: Done

File: boards/arm/stm32/photon/src/stm32_wlan.c (hunk 2 above)
Comment (anchao) [reply boards/arm/stm32/photon/src/stm32_wlan.c]: Done

**Review by xiaoxiang781216 — APPROVED** [review]

**Review by pkarashchenko — APPROVED** [review]

**Pull request merged**